deploy/sdk/README.md

```diff
@@ -77,13 +77,13 @@ app = FastAPI(title="Hello World!")
 
 
 @service(
-    dynamo={"enabled": True, "namespace": "inference"},
+    dynamo={"namespace": "inference"},
     app=app,
 )
 class Frontend:
     middle = depends(Middle)
 
-    @dynamo_endpoint(is_api=True)
```

Comment:
Can you elaborate on these changes in the PR description for future reference when looking back at this PR's commit?
1. `dynamo_endpoint` vs `dynamo_api` - when to use dynamo_api instead
2. the remove of "enabled" field? Always thought it was a little weird, so happy to see it removed, but curious on the details under the hood